src/Mod/Part/App/FeatureExtrusion.cpp

```diff
@@ -337,154 +342,316 @@ App::DocumentObjectExecReturn* Extrusion::execute(void)
 
 void Extrusion::makeDraft(const ExtrusionParameters& params, const TopoDS_Shape& shape, std::list<TopoDS_Shape>& drafts)
 {
-    double distanceFwd = tan(params.taperAngleFwd) * params.lengthFwd;
-    double distanceRev = tan(params.taperAngleRev) * params.lengthRev;
+    std::vector<std::vector<TopoDS_Shape>> wiresections;
+
+    auto addWiresToWireSections =
+        [&shape, &wiresections](std::vector<std::vector<TopoDS_Shape>>& wiresections) -> size_t {

```

Comment:
```suggestion
        [&shape](std::vector<std::vector<TopoDS_Shape>>& wiresections) -> size_t {

```
because I got the following build error:
```
FreeCAD/src/Mod/Part/App/FeatureExtrusion.cpp:348:73: error: lambda parameter ‘wiresections’ previously declared as a capture
  348 |         [&shape, &wiresections](std::vector<std::vector<TopoDS_Shape>>& wiresections) -> size_t {
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
```

blindly following my uninformed interpretation of what is said here: https://stackoverflow.com/questions/61559765/checking-presence-of-an-element-in-a-list-using-2-params-and-a-lambda-function-i
I did this change and it looks like it compiled that file, no clue if it's still doing what's intended like this since I don't even understand what lambda parameters, capture and all those things mean. No clue if it will actually finish compiling either but I want to go to sleep now and my shitty notebook will still take a long time to finish compiling so I'm just going to leave this comment now hoping it's useful.

Reply:
> because I got the following build error:

Thanks. That's why i need the CI. I use MSVC and this did not issue even a warning despite the obvious mistake "lambda parameter 'wiresections' previously declared as a capture"

I fixed this now and rebased to trigger another CI run.